### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewSearch_ColorContrastAnalyzer: Color Contrast ratio for "Search"/"Replace" placeholder is less than required in Light VS theme.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52267]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.4.3
Tool Used : Color Contrast Analyzer

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Welcome screen.
3. Apply Light Visual Studio Theme.
4. Start Color Contrast Analyzer to determine the color contrast ratio for "Search"/"Replace" placeholder.

**Actual:**
Color Contrast ratio for "Search"/"Replace" placeholder is less than 4.5:1 in Light VS theme.

**Expected:**
Color Contrast ratio for "Search"/"Replace" placeholder should be >=4.5:1 in Light VS theme.

**Recommendations:**
Adjust color of that element to meet the minimum color contrast requirement. 
Or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Low visioned users will not clearly understand the placeholder text written on the screen.

**MAS Reference:**
MAS 1.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment for Reference:**

![a11y_vscode_viewsearch_colorcontrastanalyzer_placeholder](https://user-images.githubusercontent.com/39204909/41581205-68aec2e4-73bb-11e8-9f2b-2ed4c0e5d58c.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by misolori** [comment]
Duplicate #52593
